src/app/wizard/step/cluster/component.ts

```diff
@@ -342,6 +342,8 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
     const cniPluginType = this.controlValue(Controls.CNIPlugin);
     const cniPluginVersion = this.controlValue(Controls.CNIPluginVersion);
     const cniPlugin = cniPluginType ? {type: cniPluginType, version: cniPluginVersion} : null;
+    const konnectivity = this.isKonnectivityEnabled ? this.controlValue(Controls.Konnectivity) : null;
```

Comment:
Does this mean that if `this.isKonnectivityEnabled` will be false then even if my checkbox in the UI will be marked it will have no effect? If that is the case then it will be very confusing.

If Konnectivity is disabled in general then the checkbox should be hidden or disabled and empty.

Reply:
Took it from the issue `Note: in case that the Konnectivity feature gate in KKP is not enabled, this checkbox should not be shown at all (which should already be the case) and the value should not be set in the API call.`
Atm the checkbox isn't visible in the ui if `!this.isKonnectivityEnabled`

Reply:
if the Konenctivity feature gate is disabled then I would prefer to not show the checkbox in the UI (make it hidden / not visible).
In that case it is not super-important whether the UI sends null or false to backend - both should work fine.

Reply:
I didn't see it as this was done not in this pull request. In the end this line might not be needed but it's okay.